selfdrive/car/tesla/carcontroller.py

```diff
@@ -372,7 +372,7 @@ def update(self, enabled, CS, frame, actuators, \
     elif self.PCC.enable_pedal_cruise:
       CS.v_cruise_pcm = self.PCC.pedal_speed_kph * speed_uom_kph
     else:
-      CS.v_cruise_pcm = max(0.,CS.v_ego * CV.MS_TO_KPH  +0.5) * speed_uom_kph
+      CS.v_cruise_pcm = max(0., CS.v_ego * CV.MS_TO_KPH) * speed_uom_kph
```

Comment:
I believe you need to add 0.5 at then end (after UOM multiplication) for the correct value to be shown all the time.

Reply:
Yes, I moved it to selfdrive/car/tesla/teslacan.py#159 to apply it always. Because before it only applied when ACC and PCC were not engaged.